app/assets/stylesheets/components/_navbar.scss

```diff
@@ -31,6 +35,7 @@ nav {
       width: 0;
       overflow: hidden;
       &:hover {
+
         &::before {
```

Comment:
The spacing was between `overflow` and `&:hover`, not between `&:hover` and `&::before`
Btw if you're not doing anything in `&:hover` write `&:hover::before` instead